web/templates/settings/admin_invites_list.hbs

```diff
@@ -41,13 +51,13 @@
               >
                 <i class="fa fa-paper-plane"></i>
             </button>
-        {{/unless}}
+        {{/if}}
         <button
           class="button rounded small revoke-invite btn-danger"
           {{#if disable_buttons}}disabled="disabled"{{/if}}
           data-invite-id="{{id}}"
           data-is-multiuse="{{is_multiuse}}"
-          data-tippy-content="{{t 'Revoke invite'}}"
+          data-tippy-content="{{t 'Revoke'}}"
```

Comment:
This change should also be in the commit which adds the tooltip as mentioned in the previous comment.